commit 8ac6c120395be62ea2064a929333e533cbc7ca40 (refs/changes/89/289/1)
Author: Andrey Yatskov <andrey.yatskov@silvertreesystems.com>
Date:   2016-08-15T17:42:16+03:00 (3 years, 2 months ago)
    
    PORTAL-2165 The ability to append fields to the log record

diff --git a/lib/cloudapi.js b/lib/cloudapi.js
index ffd3581..f41614d 100644
--- a/lib/cloudapi.js
+++ b/lib/cloudapi.js
@@ -192,20 +192,16 @@ function CloudAPI(options) {
     assert.optionalObject(options.logFields, 'options.logFields');
 
     if (options.logFields) {
-        options.log = log.child(options.logFields);
+        log = log.child(options.logFields);
     } else {
-        options.log = log.child();
+        log = log.child();
     }
 
     if (options.logLevel) {
-        options.log.level(options.logLevel);
+        log.level(options.logLevel);
     }
-
-    if (options.api_version)
-        options.version = options.api_version;
-
-    if (!options.version)
-        options.version = API_VERSION;
+    options.log = log;
+    options.version = options.api_version || API_VERSION;
 
     this.account = options.account || 'my';
     this.sign = options.sign;
